execution/executor-benchmark/src/pipeline.rs

```diff
@@ -164,7 +164,12 @@ where
                     info!("Received block of size {:?} to execute", block_size);
                     executed += block_size;
                     stage_executed += block_size;
-                    exe.execute_block(current_block_start_time, partition_time, block);
+                    exe.execute_block(
+                        current_block_start_time,
+                        partition_time,
+                        block,
+                        overall_measuring.event_summary_mut(),
```

Comment:
somewhat related to above comment: 
don't you want 'stage_overall_measuring' to capture events in that stage ? because we capture the measuring of every stage at line 180 and reset 'stage_overall_measuring' at 186